README.md

```diff
@@ -109,11 +109,7 @@ When `deploy`ing or `push`ing, the script will attempt to parse the latest on-wi
 Style guideline
 ---------------
 
-While old legacy code has many different and incoherent styles, it has been decided to utilize a more coherent style throughout the code.
-
-The [jQuery Core Style Guideline][jq_style] is what we will hereafter use as our style guideline.
-
-Needless to say, there are exceptions. The main sticking point is spacing around parentheses. Older Twinkle code looks like `if ( condition ) {`, but newer code tends to use `if (condition) {`. The best convention here is to follow the style of surrounding code.
+While old legacy code previously had many different and incoherent styles, it has been decided to utilize a more coherent style throughout the code, both for consistency and to cut down on potential errors.  [eslint][eslint.org] can be used to check your code before submission - try running `npm run lint` or `eslint .` in your terminal - and can even repair many common issues via `eslint . --fix`.  More information on specific style rules can be seen in [issue #500][fivehundred] and in `.eslintrc.json`, but the best advice is to just follow the style of surrounding code!
```

Comment:
By default eslint wouldn't be installed globally. I think `./node_modules/.bin/eslint` in theory should work for everyone, but the npm scripts are even better. We should add a `fix` script that runs `eslint . --fix`. Also probably worth mentioning [npm](https://www.npmjs.com/) is required (for those who are unfamiliar with it), and that you need to first run `npm install` for the linting scripts to work.

Reply:
@MusikAnimal Word.  I've got plans to write up and do a lot of that as part of #477 and beyond — including some file reorganization and a /dev folder — but I'll fix up this note here per your recommendations.

Reply:
@MusikAnimal Rewritten, will expand as part of 477.  I'll probably add a `lintfix` (`fixlint`?) shortcut then as well, but given how we define `lint` we can just run `npm run lint -- --fix` to fix things at the moment.

Reply:
:+1: